museum/sampleRecords/adminDonor.browse.bundles.json

```diff
@@ -1,5 +1,4 @@
-{"criteria":{"status_facet": [0]},
```

Comment:
See further down - I've moved from the browse service for sample data to the search service. This is because there are no browses defined for movements. It does mean that the test imports have to run with search indexing on the fly, but I'm doing that anyway.